mesheryctl/internal/cli/root/system/start.go

```diff
@@ -79,16 +84,47 @@ func ValidateComposeFileForRecreation(CurrentServices map[string]utils.Service,
 	return nil
 }
 
-func start() error {
-	if _, err := os.Stat(utils.MesheryFolder); os.IsNotExist(err) {
-		if err := os.Mkdir(utils.MesheryFolder, 0777); err != nil {
-			return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to make %s directory", utils.MesheryFolder)))
+func applyManifest(manifest []byte, client *meshkitkube.Client) error {
+	err := client.ApplyManifest(manifest, meshkitkube.ApplyOptions{
+		Namespace: "default",
+		Update:    true,
+		Delete:    false,
+	})
+
+	if err != nil {
+		return errors.Wrap(err, "failed to apply manifests")
+	}
+	return nil
+}
+
+// ApplyManifestFiles applies all the required manifests into the Kubernetes cluster
+func ApplyManifestFiles(requestedAdapters []string, client *meshkitkube.Client) error {
+	log.Info(requestedAdapters)
+	manifestFiles := filepath.Join(utils.MesheryFolder, utils.ManifestsFolder)
+	for _, adapter := range requestedAdapters {
+		adapterFile := filepath.Join(manifestFiles, adapter)
+		adapterDeployment := adapterFile + "-deployment.yaml"
+		adapterService := adapterFile + "-service.yaml"
+		log.Info(adapterDeployment)
+		manifestDepl, err := ioutil.ReadFile(adapterDeployment)
+		if err != nil {
+			return errors.Wrap(err, "failed to read manifest files")
 		}
+		manifestService, err := ioutil.ReadFile(adapterService)
+		if err != nil {
+			return errors.Wrap(err, "failed to read manifest files")
+		}
+		applyManifest(manifestDepl, client)
+		log.Info(manifestService)
 	}
+	log.Info("Applied manifests!")
+	return nil
+}
```

Comment:
Helper function to apply the manifests to kubernetes cluster. Checks for the adapters specified by the user and only applies those.